api_tests/nodes/views/test_node_comments_list.py

```diff
@@ -123,7 +123,6 @@ def _set_up_public_project(self):
     def _set_up_public_project_with_public_comment_level(self):
         """ Public project configured so that any logged-in user can comment."""
         self.project_with_public_comment_level = ProjectFactory(is_public=True, creator=self.user)
-        self.project_with_public_comment_level.comment_level = 'public'
```

Comment:
This is currently the same as `_set_up_public_project` now that `comment_level = public` by default. Instead, it should be `_set_up_public_project_with_private_comment_level`, with `comment_level = private`.  

Then you can add `test_public_node_non_contributor_cannot_comment` back in using this set-up method, since we still want to be sure non-contributors can't comment on projects when `comment_level = private` 

And you can replace any usages of `_set_up_public_project_with_public_comment_level` with `_set_up_public_project`